web/src/css/vendor-bootstrap.less

```diff
@@ -5,7 +5,7 @@
 // Reset and dependencies
 @import "../../node_modules/bootstrap/less/normalize.less";
 @import "../../node_modules/bootstrap/less/print.less";
-@import "../../node_modules/bootstrap/less/glyphicons.less";
+// @import "../../node_modules/bootstrap/less/glyphicons.less";
```

Comment:
Why is this commented out? If it's no longer in use we can delete it instead of commenting it out :)